express-basic/server.js

```diff
@@ -11,20 +11,16 @@ const app = Express();
 
 app.use("/color/:name", function (req, res) {
   let name = req.params.name;
```

Comment:
no semi colons in standardjs

Reply:
This was caused by prettier when I tried to format the code for better indentation.

Reply:
I find standard format vs extension useful for this particular task, not semi-colons at all. I should push change later on.